### ISSUE TITLE
Document type stuck with "CalConnect International Standard" phrase

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/asciidoctor-csd#24]
The "CalConnect International Standard" and "Proposal" phrases overlap.

It should probably be just "CalConnect Proposal"?

![screen shot 2018-05-16 at 4 03 21 pm](https://user-images.githubusercontent.com/11865/40104387-c6cf5eb4-5922-11e8-918b-f5118ac407fe.png)

**Comment by opoudjis** [comment]
CalConnect International Standard is too long for the space provided; shouldn't it just be "Calconnect Standard" in the sidebar? The "Proposal" element is not the issue, and no reason to rename "Calconnect Standard" to "Calconnect Proposal".

**Comment by ronaldtse** [comment]
Right, we just need to be firm on the types of document. In any case, “CalConnect International Standard” is not what we intended. “CalConnect Standard” seems like the proper one for ‘standard’ type (as opposed to the types like ‘report’ or ‘recommendation’)

**Comment by opoudjis** [comment]
"International Standard" is inherited from ISO. I'll overrule it.

**Comment by opoudjis** [comment]
... In fact, I thought I was already blocking the ISO types, but I was doing it in the wrong place in the grammar. Fixed.

If you put "standard" in the :doctype: attribute, all will be well. If you put "international-standard" in the :doctype: attribute, that is what is going to end up in the sidebar—I'm not going to abort execution because of it. But now that I've fixed the grammar, it will be flagged as an error. if you want me to ignore illegal elements in the :doctype: attribute completely, and overwrite them with a default value of "standard", let me know.
